### PR TITLE
feat: SDA-2775: Notification Position Window New UI

### PR DESCRIPTION
**Pull request by VishwasShashidhar**
## Description

- Create new user interface for notification position window
- Add unit tests

## Screenshots

Screenshot 2020-12-30 at 15 54 22

## Related PRs

N/A

### CODE REVIEW

File: src/renderer/styles/notification-settings.less [hunk 1]
```diff
@@ -3,112 +3,181 @@
...
-  text-align: center;
-  white-space: nowrap;
-  overflow: hidden;
-  width: 100%;
-  color: @color;
+  text-align   : center;
+  white-space  : nowrap;
+  overflow     : hidden;
+  width        : 100%;
+  color        : @color;
```
Comment (KiranNiranjan) [inline src/renderer/styles/notification-settings.less:44]: Is this spacing required?

**Review by KiranNiranjan — APPROVED** [review]
LGTM 👍🏻

**Comment by NavyaGopalakrishna** [comment]
LGTM 😃

**Review by NavyaGopalakrishna — APPROVED** [review]

File: src/renderer/styles/notification-settings.less (hunk 1 above)
Comment (VishwasShashidhar) [reply src/renderer/styles/notification-settings.less:44]: Oh, it looks like the formatter has included that :) I'll fix it.

**Pull request merged**